fennel/featuresets/featureset.py

```diff
@@ -382,6 +471,19 @@ def _add_feature_names_as_attributes(self):
 
     def _get_extractors(self) -> List[Extractor]:
         extractors = []
+        # auto generated extractors for features
+        for feature in self._features:
+            if feature.extractor:
+                extractor = feature.extractor
+                if extractor.extractor_type == ExtractorType.LOOKUP and (
+                    extractor.inputs is None or len(extractor.inputs) == 0
+                ):
+                    feature.extractor.set_inputs_from_featureset(self)
+                # TODO TODO zaki decide if we want this here or not: should we actually capture
+                # these as featureset extractors?
+                extractors.append(extractor)
```

Comment:
You need to right? So that validate can check using all extractors defined?

Reply:
Yeah I am, just forgot to remove the comment :)